lib/fortnox/api/types.rb

```diff
@@ -38,6 +38,7 @@ module Types
       AccountNumber = Strict::Int
                       .constrained(gteq: 0, lteq: 9999)
                       .optional
+                      .constructor(Dry::Types::Coercions::Form.method(:to_int))
```

Comment:
I like the thought behind this. You can probably achieve the same result by changing `Strict::Int` to `Coercible::Int` at the top of the declaration. I'm assuming this doesn't work for the empty string case though.

To correctly deal with nil and empty string you can do something like this:

```
AccountNumber = Strict::Int
  .constrained(gteq: 0, lteq: 9999)
  .optional
  .constructor { |v| Integer(v) unless v.nil? or v.empty? }
```